docs/gettingstarted/index.rst

```diff
@@ -159,6 +159,9 @@ Getting started with Flocker
          .. version-code-block:: console
 
             you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/Vagrantfile && \
+              curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/cluster.crt && \
```

Comment:
One way to deal with it is as @exarkun mentioned on another branch - have one `curl` for a compressed file.